### PR TITLE
building bridges with DragonflyBSD, and NetBSD

### PR DESCRIPTION
**Pull request by egypcio**
**DragonflyBSD**
  - dfbsd is based on FreeBSD, and still keeps its default settings/paths;
  - packages names and most configurations are just the same as fbsd;
  - obfs4 is available! we can have more diverse bridges running.

**NetBSD**
  - of course it runs NetBSD ]:-)

### CODE REVIEW

File: content/relay-operations/technical-setup/bridge/dragonflybsd/contents.lr [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+```
+pkg install tor-devel
+
+### 3. Configure `/usr/local/etc/tor/torrc` to run **Tor** as a Bridge
+
+```
+RunAsDaemon 1
+BridgeRelay 1
+
+# Replace "TODO1" with a Tor port of your choice.  This port must be externally
```
Comment (mpguerra) [inline content/relay-operations/technical-setup/bridge/dragonflybsd/contents.lr]: This is somehow not rendering properly... it seems it's not recognised as a code block, but a title instead... and it messes up the rest of the content further down

**Comment by mpguerra** [comment]
NetBSD section looks good :)

File: content/relay-operations/technical-setup/bridge/netbsd/contents.lr [hunk 2]
```diff
@@ -0,0 +1,96 @@
+_model: page
+---
+color: primary
+---
+title: NetBSD
+---
+html: two-columns-page.html
+---
+key: 3
```
Comment (NullHypothesis) [inline content/relay-operations/technical-setup/bridge/netbsd/contents.lr]: Same as above: not sure what `key` is supposed to be but another guide is already using it.

File: content/relay-operations/technical-setup/bridge/netbsd/contents.lr [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+```
+ln -sf /usr/pkg/share/examples/rc.d/tor /etc/rc.d/tor
+echo "tor=YES" >> /etc/rc.conf
+/etc/rc.d/tor start
+```
+
+### 4. Monitor your logs
+
+To confirm your bridge is running with no issues, you should see something like this:
```
Comment (NullHypothesis) [inline content/relay-operations/technical-setup/bridge/netbsd/contents.lr]: Maybe mention how the operator can check their syslog?

**Comment by egypcio** [comment]
hey hey! hi @mpguerra and @NullHypothesis ,
I did update the contents here and we should be fine so far. can you also give me a hand and see if passes on your side?

  - consistency added for dragonfly and netbsd;
  - both setups have information on how to check its log outputs;
  - new key id for both pages;
  - obfs4proxy-tor pkg added to right location this time (dflybsd).

many thanks for all nice inputs! appreciated

**Comment by NullHypothesis** [comment]
The changes look good to me. Thanks, @egypcio! I just merged your two new guides.

**Pull request merged**